src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,376 @@
+The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example:
+Three brain slices (`sample-01` to `sample-03`) extracted from subject `sub-01`.
+```Text
+sub-01/
+    microscopy/
+        sub-01_sample-01_<modality_suffix>.<ext>
+        sub-01_sample-02_<modality_suffix>.<ext>
+        sub-01_sample-03_<modality_suffix>.<ext>
+        sub-01_<modality_suffix>.json
+```
+In this example, JSON metadata is common for all samples of `sub-01`.
+JSON metadata may be defined per subject or per sample as appropriate, per the
+[inheritance principle](../02-common-principles.md#the-inheritance-principle).
+
+The [`acq-<label>`](../99-appendices/09-entities.md#acq) entity corresponds to a custom label that
+MAY be used to distinguish a different set of parameters used for acquiring the same modality.
+For example, two images of the same sample acquired with different magnification of 40x and 60x.
+In such case two files could have the following names:
+`sub-01_sample-01_acq-40x_<modality_suffix>.<ext>` and
+`sub-01_sample-01_acq-60x_<modality_suffix>.<ext>`, however the user is free to choose any other
+label as long as they are consistent across subjects and sessions.
+
+The [`stain-<label>`](../99-appendices/09-entities.md#stain) entity can be used to distinguish
+image files from the same sample using different stains or antibodies for contrast enhancement.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example:
+Brain slice (`sample-01`) extracted from subject `sub-01`, imaged with three stains
+(`stain-01`, `stain-02`, `stain-03`) in three separate files.
+In this example, the entity stain is used to distinguish images with different
+stains in separate files from the same sample.
+```Text
+sub-01/
+    microscopy/
+        sub-01_sample-01_stain-01_<modality_suffix>.<ext>
+        sub-01_sample-01_stain-01_<modality_suffix>.json
+        sub-01_sample-01_stain-02_<modality_suffix>.<ext>
+        sub-01_sample-01_stain-02_<modality_suffix>.json
+        sub-01_sample-01_stain-03_<modality_suffix>.<ext>
+        sub-01_sample-01_stain-03_<modality_suffix>.json
+```
+In this example, the entity stain is used to distinguish images with different
+stains in separate files from the same sample.
+In the case where a single file contains different staining in each channel, the
+`stain-<label>` is omitted.
+
+Stains or antibodies SHOULD be indicated as appropriate in the `"SampleStaining"`,
+`"SamplePrimaryAntibodies"` and/or `"SampleSecondaryAntobodies"` keys in the sidecar JSON file,
+although the label may be different.
+
+If more than one run of the same sample, acquisition and stain are acquired during the same
+session, the [`run-<index>`](../99-appendices/09-entities.md#run) entity MUST be used:
+`_run-1`, `_run-2`, `_run-3`, and so on.
```

Comment:
I'm always wondering when indexes should be zero-padded. 
Here it says it's recommended: 
https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#participant-names-and-other-labels
This often raises some questions at the time of developing apps. Maybe here we can zero-pad in the example?

```suggestion
`_run-01`, `_run-02`, `_run-03`, and so on.
```

Reply:
That is a good question, I was wondering the same while writing this. :)
In this case, I used the same kind of padding as in the MRI and PET examples for uniformity.

Reply:
Sounds very good!